integrations/slack/src/router.ts

```diff
@@ -83,6 +111,29 @@ export const handleFetchEvent: FetchEventCallback = async (request, context) =>
         });
     });
 
+    router.get('/spaces', async () => {
+        let spaces: { value: string; label: string }[];
+
+        if (!('organization' in environment.installation.target)) {
+            spaces = [];
+        } else {
+            const resp = await api.orgs.listSpacesInOrganizationById(
+                environment.installation.target.organization
+            );
+
+            spaces = resp.data.items.map((space) => ({
+                value: space.id,
+                label: space.title,
+            }));
+        }
+
+        return new Response(JSON.stringify(spaces), {
+            headers: {
+                'Content-Type': 'application/json',
+            },
+        });
+    });
+
```

Comment:
```suggestion
```

Reply:
This section was added by Steven. Is this specific to the save stuff?

Reply:
Yes, we can remove.